core/src/main/java/com/google/instrumentation/stats/Stats.java

```diff
@@ -19,17 +19,17 @@
  * {@link Stats}.
  */
 public final class Stats {
-  private static final StatsContextFactory CONTEXT_FACTORY = Provider.newInstance(
-      "com.google.instrumentation.stats.StatsContextFactoryImpl", null);
-
   private static final StatsManager STATS_MANAGER = Provider.newInstance(
```

Comment:
This is not a constant:
 https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

I think the name should be statsManager